pkg/cmd/cli/cmd/importimage.go

```diff
@@ -383,7 +383,7 @@ func (o *ImportImageOptions) importTag(stream *imageapi.ImageStream) (*imageapi.
 
 	} else {
 		// create a new tag
-		if len(from) == 0 {
+		if len(from) == 0 && tag == imageapi.DefaultImageTag {
```

Comment:
Nope, the tag will always be initiated to latest, if not provided in [here](https://github.com/openshift/origin/blob/f0b6956926e26aa5daf4452d927379f02b0ac82e/pkg/cmd/cli/cmd/importimage.go#L120), `tag` is assigned from `o.Tag` at the beginning of `importTag` function.